app/events/rtjc.go

```diff
@@ -80,7 +96,7 @@ func (l Rtjc) Listen(ctx context.Context) {
 			if serr := l.Submitter.Submit(ctx, msg, pin); serr != nil {
 				log.Printf("[WARN] can't send message, %v", serr)
 			}
-			sendSummary(msg)
+			go sendSummary(msg)
```

Comment:
i don't like it at all. The idea of starting potentially infinite and totally uncontrolled number of goroutines in the situation where nothing managing their lifecycle - it is troublesome idea. To achieve the same effect in much more manageable manner you better send this msg to some buffered channel with select on full capacity to avoid blocking. And you may have a pool of goroutines in some way, or alternatively using limited concurrency primitives, like those i have in go-pkgz/syncs

Reply:
I don't like it too, but you're concerned about execution time for my `sleep` method in the cycle.
I'm thinking of sending summary messages in `openai` package. Here I'm going to keep the same logic which was here before your changes by adding summary + pushing link/message for summary in the channel.

Again, this is feature for next PR.